README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# HW1
+
+# About me
+My name is **Chenyang He**. I am a MAPSS student concentrating in *Psychology*. 
+
+# Research Interest 
+I am currently working as an RA in Dr. Boaz Keysar's Multilingualism and Decision-Making Lab in the Psychology Department. 
+Here's a link to our lab page:
+[mdl](https://mdl.uchicago.edu/)
+
+We are dedicated to understanding the following interesting topics:
+* Bilingualism and Decision Making
+  * With millions of people worldwide using a foreign language everyday, how does language impact the way they make decisions? 
+* Accents
+  * What are the consequences of speaking with a foreign accent? 
+* Figurative Language
+  * How do people understand figurative language such as metaphors, idioms, irony and sarcasm? 
+
+# My best friend
+This is Peaches, my best friend during the quarantine.
+She makes me happy everyday :)
+![peaches](https://github.com/sarahcy1/hw01/blob/master/IMG_4007.jpg)
```

Comment:
The picture of your cat is also adorable! I like how it's well-sized to take up the width of the page exactly, and the angle makes me feel like she's looking at me.